Content.Shared/_RMC14/TacticalMap/TacMapMarineAlertSystem.cs

```diff
@@ -19,6 +22,7 @@ public override void Initialize()
 
         SubscribeLocalEvent<TacMapMarineAlertComponent, MapInitEvent>(OnMapInit);
         SubscribeLocalEvent<TacMapMarineAlertComponent, ComponentRemove>(OnRemove);
+        SubscribeLocalEvent<TacMapMarineAlertComponent, MoveEvent>(OnMove);
```

Comment:
Having this on MoveEvent would be extremely expensive for all marines and xenos, it could be better to do it only for players every x amount of time, and adding a limit to the update time, like intel system does